ui/src/components/Readme/__snapshots__/Readme.test.tsx.snap

```diff
@@ -23,10 +23,7 @@ exports[`should render the readme file 1`] = `
             #
           </span>
           <span className=\\"token title\\" style={{...}}>
-             
-          </span>
-          <span className=\\"token code\\" style={{...}}>
-            &#96;az&#96;
+             &#96;az&#96;
```

Comment:
Does the above dependency change the snapshot ??

Reply:
yes